agent/tests/test_user_instrument.c

```diff
@@ -52,14 +52,124 @@ static void test_op_array_wraprec(TSRMLS_D) {
   tlib_php_request_end();
 }
 
+static void test_get_wraprec_by_name() {
+#if ZEND_MODULE_API_NO < ZEND_7_3_X_API_NO
```

Comment:
Elsewhere it seems `nr_php_get_wraprec_by_name()` is used for PHP 8.0 and newer - is there a reason this test is run for PHP 7.4?

Reply:
We will only use this for oapi (so php 8.0+), but the functionality is valid for other PHPs, so the logic was if it works for other PHPs, why not test for it.